OpenRA.Game/ObjectCreator.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace OpenRA
 {
-	public class ObjectCreator
+	public class ObjectCreator : IDisposable
```

Comment:
Nit: `sealed` here due to `IDisposable` (or ` protected void Dispose(bool disposing)` so any theoretical derived classes can hook into it).